CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -185,6 +185,51 @@ public let EXPR_NODES: [Node] = [
     ]
   ),
 
+  // the canImport expr in if config expression
+  Node(
+    name: "CanImportExpr",
+    nameForDiagnostics: "'canImport' expression in if config expression",
```

Comment:
I’d just change this to the following. It’s a bit more concise and doesn’t make as many assumptions about where `CanImportExpr` could also be used.

```suggestion
    nameForDiagnostics: "'canImport' expression",
```